docs/config.md

```diff
@@ -82,6 +82,7 @@ The exporter is used to output the telemetry.
 | `OTEL_EXPORTER_OTLP_TIMEOUT` | Maximum time the OTLP exporter will wait for each batch export. | `1000` (ms) |
 | `OTEL_EXPORTER_OTLP_PROTOCOL` | The OTLP expoter transport protocol. Supported values: `grpc`, `http/protobuf`. [1] | `http/protobuf` |
 | `OTEL_EXPORTER_ZIPKIN_ENDPOINT` | Zipkin URL. | `http://localhost:8126` |
+| `OTEL_DOTNET_AUTO_HTTP2UNENCRYPTEDSUPPORT_ENABLED` | Allows to enable `System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport`. It is required by OTLP over `grpc` exporter execute on .NET Core 3.x. For details plesae check [this documentation](https://docs.microsoft.com/en-us/aspnet/core/grpc/troubleshoot?view=aspnetcore-6.0#call-insecure-grpc-services-with-net-core-client) | `false` |
```

Comment:
```suggestion
| `OTEL_DOTNET_AUTO_HTTP2UNENCRYPTEDSUPPORT_ENABLED` | Enables HTTP2 unencrypted support. Required by OTLP over gRPC when instrumenting NET Core 3.x applications. See [the official Microsoft documentation](https://docs.microsoft.com/en-us/aspnet/core/grpc/troubleshoot?view=aspnetcore-6.0#call-insecure-grpc-services-with-net-core-client) for more details. | `false` |
```

Reply:
Agreed with @theletterf  that 94d3b2c9438e7e82b2c0514e01fe07c85d5322bb is fine.